### PR TITLE
Remove custom font styles for landing page alert and intro text

### PR DESCRIPTION
**Pull request by jzucadi**
## Summary

The blue "Learn more..." expandable alert has varying font sizes and styles that differ from the VA design system. This PR removes the custom styles.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#88857

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   Screenshot 2024-07-30 at 4 13 50 PM |  Screenshot 2024-07-30 at 4 07 34 PM |
| Desktop |  Screenshot 2024-07-30 at 4 14 10 PM |  Screenshot 2024-07-30 at 4 07 10 PM|

## What areas of the site does it impact?

My HealtheVet Landing page

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv-landing-page/components/HeaderLayout.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const HeaderLayout = ({
               </div>
             </div>
           </div>
-          <div className="va-introtext">
```
Comment (dcloud) [inline src/applications/mhv-landing-page/components/HeaderLayout.jsx:70]: I didn't expect to see changes to the intro text based on the ticket. Couldn't we just move the `<va-alert-expandable />` outside of the va-introtext div?

**Review by dcloud — DISMISSED** [review]
This changes the introtext style, which is not what the ticket indicates needs changing

File: src/applications/mhv-landing-page/components/HeaderLayout.jsx (hunk 1 above)
Comment (jzucadi) [reply src/applications/mhv-landing-page/components/HeaderLayout.jsx]: yes, will do

**Comment by jzucadi** [comment]
> This changes the introtext style, which is not what the ticket indicates needs changing

How this ticket was originally worded when I worked on it indicated that there were styles problems on the landing page and the expandable was just a particular example. I see now that the language has been updated to be more focused. I've removed the style change done to the intro text.

**Review by Andrew565 — APPROVED** [review]

**Review by dcloud — APPROVED** [review]
Looks good!

**Pull request merged**